### DISCUSSION TITLE
Add uuid compatibility on team_id

### DISCUSSION THREAD

**Discussion by juarezfranco** [spatie/laravel-permission#1854] [Q&A] [answered]
In `Spatie\Permission\PermissionRegistrar` add uuid compatibility `$teamId`,  `setPermissionsTeamId` and `getPermissionsTeamId`.

is it possible?

**Comment by erikn69** [comment, marked as answer]
Is posible, removing data type on that methods, can you test that way? if works i will make a PR for that

**Reply by juarezfranco** [reply]
Yes it works, I'm doing it.
